java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -2342,6 +2389,23 @@ public CompletableFuture<Map<String, String[][]>> xclaim(
         return commandManager.submitNewCommand(XClaim, args, this::handleMapResponse);
     }
 
+    @Override
+    public CompletableFuture<Map<GlideString, GlideString[][]>> xclaim(
+            @NonNull GlideString key,
+            @NonNull GlideString group,
+            @NonNull GlideString consumer,
+            long minIdleTime,
+            @NonNull GlideString[] ids,
+            @NonNull StreamClaimOptions options) {
```

Comment:
the consumer can be glide string no?

Reply:
It is GS, or I didn't understand your comment